packages/alignments/src/AlignmentsTrack/configSchema.ts

```diff
@@ -33,6 +33,11 @@ const configModelFactory = (pluginManager: PluginManager) => {
         description: 'maximum value for the SNP coverage y-scale',
         defaultValue: Number.MAX_VALUE,
       },
+      maxViewBpPerPx: {
+        type: 'number',
+        description: 'maximum bpPerPx that is displayed in the view',
+        defaultValue: 100,
+      },
```

Comment:
I think this name might be a bit confusing, but I'm also not sure what it should be. Maybe `maxViewableBpPerPx` or `maxDisplayedBpPerPx`?

Reply:
changed to maxDisplayedBpPerPx, I like that name better